### ISSUE TITLE
Preview won't work if basePath: is set

### ISSUE THREAD

**Issue by razirazo** [dzylikecode/VSCodeExt-docsify-Preview#23]
I want to host my documentation in GitHub Pages, so I have to set basePath: to point it to my repo pages.
Now, instead of rendering my locally edited copy, the index html is retrieving the remote files hosted in github.

Is there any way to override (or remove) basePath: in vscode preview?

**Comment by dzylikecode** [comment]
Maybe you can create a new HTML file named `indexLocal.html` whose `basePath` is not set, and set it as the Docsify entry: `/docs/indexLocal.html`. Don't forget to add it to the `.gitignore`. 😃

**Comment by dzylikecode** [comment]
> I want to host my documentation in GitHub Pages, so I have to set basePath: to point it to my repo pages.

I think it works though you don't set the `basePath`.  `basePath` is used where the files are placed on the other site.

**Comment by razirazo** [comment]
> > I want to host my documentation in GitHub Pages, so I have to set basePath: to point it to my repo pages.
> 
> I think it works though you don't set the `basePath`. `basePath` is used where the files are placed on the other site.

For Github Pages apparently it works only if you have a single file.
With multiple pages and _sidebar.md in use, without basePath set it will only retrieve single README.md
I'm not so sure of what's going on, but someone had the same problem before and basePath suggestion fixed it.
https://github.com/docsifyjs/docsify/issues/1623

I have tried your suggestion before, making index.html into an alternative local copy and set the new entry file. But it broke the entire site, causing a page not found error. I think 'index.html' is a hardcoded reference in Docsify, so it will not recognize the other file names.

**Comment by dzylikecode** [comment]
But it works well for me.

https://user-images.githubusercontent.com/71511961/201515890-4929f55e-4452-44e7-8ef5-cc69de5d93fa.mp4

Here are the settings.

| .vscode | `local.html`|
| :--: | :--: |
|![image](https://user-images.githubusercontent.com/71511961/201516010-de3f6d1d-f751-4ca3-8723-26279412c97c.png)| ![image](https://user-images.githubusercontent.com/71511961/201516024-97901cc5-b582-4f74-ae71-e46e551880d6.png)|

**Comment by dzylikecode** [comment]
Is there anything wrong with your blog? I have removed the `basePath` and hosted it in GitHub Pages. And all look well.

https://dzylikecode.github.io/docs.kupacareer.com.my/#/

**Comment by razirazo** [comment]
> Is there anything wrong with your blog? I have removed the `basePath` and hosted it in GitHub Pages. And all look well.
> 
> https://dzylikecode.github.io/docs.kupacareer.com.my/#/

Wow that's weird. I tried it again, and it somehow.. worked without anything else changed?
And even the local.html, no more 404 somehow.

I swear I've been pulling my hair over a week dealing with this, going as far deleting and remaking the repo from other accounts thinking I messed up the setting somewhere before I found the basepath supposed fix.

I guess it is sorted out now. Thanks for helping!
